website/docs/stitch-combining-schemas.md

```diff
@@ -97,48 +94,112 @@ export interface SubschemaConfig {
+### Merged types
 
-_**For pre-version 5:** the old method of using [makeRemoteExecutableSchema](/docs/remote-schemas/) to create a local proxy of a remote schema still works. However, it adds an additional layer of delegation that can be avoided by sending settings directly to `stitchSchemas` via SubschemaConfig._
+Duplicate type names are merged by default in GraphQL Tools v7. That means objects, interfaces, and input objects with the same name will have their fields consolidated from across subschemas, and unions will consolidate all member types. The combined gateway schema will then smartly delegate portions of a request to the proper origin schema(s). See [type merging guide](/docs/stitch-type-merging/) for a comprehensive overview.
 
-## Duplicate type definitions
+Type merging will only encounter conflicts on fields and type-level descriptions. By default, the final definition of a field or type description found in the subschemas array is used. You may customize this with `typeMergingOptions` selection logic. For example, the following handlers will select the first non-blank description for each type and field encountered in the subschemas array:
+
+```js
+const gatewaySchema = stitchSchemas({
+  subschemas: [...],
+  mergeTypes: true, // << optional in v7
+  typeMergingOptions: {
+    typeDescriptionsMerger(candidates) {
+      const candidate = candidates.find(({ type }) => !!type.description) || candidates.pop();
+      return candidate.type.description;
+    },
+    fieldConfigMerger(candidates) {
+      const configs = candidates.map(c => c.fieldConfig);
+      return configs.find(({ description }) => !!description) || configs.pop();
+    },
+    inputFieldConfigMerger(candidates) {
+      const configs = candidates.map(c => c.inputFieldConfig);
+      return configs.find(({ description }) => !!description) || configs.pop();
+    }
+  },
+});
+```
+
+### Binary types
+
+By setting `mergeTypes: false`, only the final set of fields, arguments, and descriptions for a type found in the subschemas array will be used. Conflict resolution may be customized with an `onTypeConflict` handler:
+
+```js
+const gatewaySchema = stitchSchemas({
+  subschemas: [...],
+  mergeTypes: false,
+  onTypeConflict: (left, right, info) => {
+    return info.left.schema.version >= info.right.schema.version ? left : right;
+  }
+});
+```
 
-By default, schema stitching will override type definitions that are duplicated across subschemas&mdash;always favoring the final definition of fields, arguments, and descriptions for a type found in the `subschemas` array. This works fine when subschemas implement identical versions of an object type. For divergent type definitions, you may now enable [type merging](/docs/stitch-type-merging) (as of GraphQL Tools 5) to smartly merge partial type definitions from across subschemas.
+Practically speaking, binary resolution requires subschemas to implement identical versions of a type for parity.
```

Comment:
I'm not sure what you mean. I think custom resolvers would have to be used on the gateway to do some custom stuff, but I'm not sure. it sounds like you are saying in this sentence that binary type resolution does not really work because the two options in the binary choice will have to be identical... I'm not sure if that's accurate, it's better to say that it requires custom resolvers on the gateway?

note that overall it's not actually a binary choice, because you can return any type not just one of the two options that are passed in...

Reply:
Fair points. I've never used `onTypeConflict` myself, and I suspect saying less about it is more accurate. Simplified this whole paragraph in fa5cfda, and rewrote around a very open-ended premise of "manual conflict resolution".